spel/scripts/amigen8-build.sh

```diff
@@ -394,6 +396,21 @@ function ComposeDiskSetupString {
         DISKSETUPCMD+="-B ${AMIGENBOOTSIZE} "
     fi
 
+    # Set the size of the /boot filesystem
+    if  [[ -f /etc/oracle-release ]] ||
```

Comment:
You can actually just check the `BUILDER` env if you want... See the `case` statement on line 77...

```
BUILDER=ol-8
```

Reply:
Ok. Looks like I didn't scope-limit `BUILDER`, so I'll see if I can leverage _that_.

Reply:
Yeah, `BUILDER` is meant to be available essentially as a "global", throughout the whole script...

Reply:
Ok. Switching to that still resulted in a good build. Marking this thread resolved.